sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/MemberInitializerRedundantTest.cs

```diff
@@ -31,7 +32,10 @@ public class MemberInitializerRedundantTest
         [TestCategory("Rule")]
         public void MemberInitializerRedundant()
         {
-            Verifier.VerifyAnalyzer(@"TestCases\MemberInitializerRedundant.cs", new MemberInitializerRedundant());
+            Verifier.VerifyAnalyzer(@"TestCases\MemberInitializerRedundant.cs",
+                new MemberInitializerRedundant(),
+                ParseOptionsHelper.FromCSharp8,
+                additionalReferences: NuGetMetadataReference.NETStandardV2_1_0);
```

Comment:
This `additionalReferences` is not necessary here since we upgarded the lang version of the unit test project to 8 (#2886).
There is even a ticket to remove this logic now that we can (#2802).